src/web/lib/hakuneko/frontend@classic-dark/mangas.html

```diff
@@ -261,12 +273,92 @@
+                    //console.debug(`Check manga: ${manga.title} with connector ${manga.connector.label} initialized: ${manga.connector.initialized} - ${manga.connector.isUpdating}`);           
+                    await manga.getChapters( ( error, chapters ) => {
+                        //Is the first chapter on the top of the list marked ? If not, we have new content !
+                        manga.hasUpdate=false;
+                        if( ! Engine.ChaptermarkManager.isChapterMarked(chapters[0],Engine.ChaptermarkManager.getChaptermark( manga))){
+                            console.log('New chapter for '+manga.title);
+                            manga.hasUpdate=true;
+                            newContent.push(manga);
+                        }       
+                    });
+                    context.set('nbContentUpdating',--nbContentUpdating);
+                    return Promise.resolve(manga); 
+                };
+                context.set('nbContentUpdating','init');
+                //Init all connectors
+                Promise.all(connectors.map(initConnector))
+                .then(() => {
+                    //Checking mangas changes
+                    return Promise.all( this.mangaList.map(checkManga));
+                })
+                .then(() => {
+                    //Do we have new content ?
+                    if (!newContent.length == 0){
+                        let contentShort=[];
+                        for (var i = 0; i < newContent.length; i++) {
+                            //Prepare notificiiton text (new)
+                            contentShort.push(newContent[i].title);
+                            //Notify dom update
+                            //Find the ID
+                            let index = this.mangaList.findIndex( ( item ) => {
+                                // mangas may be different objects (synchronizing connector list) but still be equivalent
+                                // => comparing ids instead of comparing the objects directly
+                                return ( item.id === newContent[i].id );
+                            });
+                            if( index > -1 ) {
+                                this.notifyPath( 'mangaList.' + index + '.hasUpdate' );
+                            }
+                        }
+                       //Send Notification
+                        console.log(`${newContent.length} New content: ${contentShort}`); 
+                        let myNotification = new Notification('Hakuneko', {
+                            body: 'New content : '+contentShort.join(',').substring(0,185)
+                        })
+                    }
+                    this.set('updatingContent', false );
+                    this.set('nbContentUpdating',0);
+                })
+                .catch(e => console.log(`Error checking new chapters  ${e}`)); // Catch the error
```

Comment:
Use `console.warn` or `console.error`, should there be a message for the user as well?